solidity/dashboard/src/utils/ethereum.utils.js

```diff
@@ -43,3 +45,19 @@ export const getEventsFromTransaction = async (contractToEventName, txHash) => {
 
   return events
 }
+
+export function displayEthAmount(
```

Comment:
I can't find an invocation of this method with parameters. Do you think we need these parameters here?